src/TallPreset.php

```diff
@@ -2,18 +2,18 @@
 
 namespace LaravelFrontendPresets\Tall;
 
-use Illuminate\Filesystem\Filesystem;
 use Illuminate\Support\Arr;
 use Laravel\Ui\Presets\Preset;
+use Illuminate\Filesystem\Filesystem;
 
 class TallPreset extends Preset
 {
     const NPM_PACKAGES_TO_ADD = [
         '@tailwindcss/custom-forms' => '^0.2',
-        '@tailwindcss/ui' => '^0.1',
-        'alpinejs' => '^2.0',
-        'laravel-mix-tailwind' => '^0.1.0',
-        'tailwindcss' => '^1.4',
+        '@tailwindcss/ui'           => '^0.1',
+        'alpinejs'                  => '^2.0',
+        'laravel-mix-tailwind'      => '^0.1.0',
+        'tailwindcss'               => '^1.4',
```

Comment:
I'm not a massive fan of this formatting as it can be a pain to keep the values in line if the keys change. Open to other viewpoints about this though.